### PR TITLE
feat: not start from zero when create wallet

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/controllers/wallets/index.ts [hunk 1]
```diff
@@ -72,6 +72,7 @@ export default class WalletsController {
       name,
       password,
       mnemonic,
+      isImport: true,
```
Comment (ashchan) [inline packages/neuron-wallet/src/controllers/wallets/index.ts]:
```suggestion
      isImporting: true,
```

Comment (classicalliu) [reply packages/neuron-wallet/src/controllers/wallets/index.ts]: `isImport` means this wallet is generate by import or create,  I'm not sure `isImporting` is better to describe this ?

Comment (classicalliu) [reply packages/neuron-wallet/src/controllers/wallets/index.ts]: `isImport` means this wallet is generate by import or create,  I'm not sure `isImporting` is better to describe this ?

Comment (ashchan) [reply packages/neuron-wallet/src/controllers/wallets/index.ts]: Then perhaps `isImported` is more appropriate.

Comment (classicalliu) [reply packages/neuron-wallet/src/controllers/wallets/index.ts]: `isImported` is more like the `import` action done or not, what do you think?

Comment (classicalliu) [reply packages/neuron-wallet/src/controllers/wallets/index.ts]: Renamed to `isImporting`

File: packages/neuron-wallet/src/services/addresses.ts [hunk 2]
```diff
@@ -5,9 +5,15 @@ import LockUtils from 'models/lock-utils'
 import AddressDao, { Address as AddressInterface } from 'database/address/dao'
 import env from 'env'
 import AddressEntity, { AddressVersion } from 'database/address/entities/address'
+import AddressCreatedSubject from 'models/subjects/address-created-subject'
 
 const MAX_ADDRESS_COUNT = 30
 
+export interface AddressWithWay {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/services/addresses.ts]: The name is confusing, how about `AddressAndIsImporting`

Comment (classicalliu) [reply packages/neuron-wallet/src/services/addresses.ts]: `isImporting` moves to `Address` structure, so this struct deleted

**Pull request merged**